xbmc/utils/StringUtils.cpp

```diff
@@ -759,6 +759,61 @@ std::vector<std::string> StringUtils::Split(const std::string& input, const char
   return results;
 }
 
+std::vector<std::string> StringUtils::Split(const std::vector<std::string> &input, const std::vector<std::string> &delimiters, unsigned int iMaxStrings /* = 0 */)
```

Comment:
IF it's unused you should remove it.. Imho is better to have a name distinction between a regular split and this complex split..